app/assets/stylesheets/custom/_drops.scss

```diff
@@ -26,8 +26,5 @@
   border-radius: 20px;
 }
 
-.glyphicon {
-  width: 12px;
-  color: #f50;
-}
+
 
```

Comment:
I'd vote for another tiny whitespace cleanup and drop the unused white lines at the end 
![gif-fantasia-broom2](https://cloud.githubusercontent.com/assets/3685123/17300091/395e19da-5811-11e6-9bb2-f381583690aa.gif)